### PR TITLE
updated vendor deps

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 package: github.com/giantswarm/flannel-operator
 import:
 - package: github.com/giantswarm/apiextensions
-  version: clientset
-- package: github.com/giantswarm/flanneltpr
```
Comment (xh3b4sd) [inline glide.yaml]: We get rid of this now.

File: glide.yaml [hunk 2]
```diff
@@ -17,9 +15,16 @@ import:
...
+  version: kubernetes-1.8.5
+- package: k8s.io/client-go
+  version: kubernetes-1.8.5
 
-- package: github.com/coreos/etcd
-  version: 61fc123e7a8b14a0a258aa3f5c4159861b1ec2e7
-- package: golang.org/x/net
-  version: f2499483f923065a842d38eb4c7f1927e6fc6e6d
+- package: github.com/coreos/go-semver
+  version: 8ab6407b697782a06568d4b7f1db25550ec2e4c6
```
Comment (xh3b4sd) [inline glide.yaml]: All this is some weirdness I hope to get rid of when moving to use `dep` in the next step. Glide was also retarded here. It gave me some code from some commit from 2015. This hash here is the commit of the latest tag `v0.2.0`. :table-flip:.

File: service/resource/legacyv2/job.go [hunk 3]
```diff
@@ -2,9 +2,9 @@ package legacyv2
 
 import (
 	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	batchv1 "k8s.io/api/batch/v1"
+	apiv1 "k8s.io/api/core/v1"
 	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	apiv1 "k8s.io/client-go/pkg/api/v1"
-	batchv1 "k8s.io/client-go/pkg/apis/batch/v1"
```
Comment (xh3b4sd) [inline service/resource/legacyv2/job.go]: All the imports.

File: service/framework.go [hunk 4]
```diff
@@ -249,23 +247,11 @@ func newCRDFramework(config Config) (*framework.Framework, error) {
...
-			return watcher, nil
-		}
-	}
-
 	var newInformer *informer.Informer
 	{
 		c := informer.DefaultConfig()
 
-		c.WatcherFactory = newWatcherFactory
+		c.Watcher = clientSet.CoreV1alpha1().FlannelConfigs("")
```
Comment (xh3b4sd) [inline service/framework.go]: Bam.

Comment (MarcelMue) [reply service/framework.go]: I like this.

Comment (JosephSalisbury) [reply service/framework.go]: Bam!

**Pull request merged**